client/web/src/monitoring/datadog/datadogClient.ts

```diff
@@ -0,0 +1,23 @@
+import { ErrorInfo } from 'react'
+
+// Ensures consistent `ErrorContext` across `addError` calls.
+// https://docs.datadoghq.com/real_user_monitoring/browser/modifying_data_and_context/?tab=npm#enrich-and-control-rum-data
```

Comment:
```suggestion
/**
 * Ensures consistent `ErrorContext` across `addError` calls.
 * https://docs.datadoghq.com/real_user_monitoring/browser/modifying_data_and_context/?tab=npm#enrich-and-control-rum-data
 */
```

For hover definitions, especially as we export a func that uses this